src/devices/Llc200d3sh/DetectionType.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Iot.Device.Llc200d3sh
```

Comment:
- copyright header missing
- namespace should be type of device and not the device name

Reply:
considering this device works same as any other liquid switch I suggest to make a generic type like WaterLevelSwitch or similar with options to flip how `1` should be interpreted as

Reply:
Would you have the namespace as LiquidLevelSwitch or as LiquidLevel. If you used LiquidLevel then in theory you could also include sensors that measure the volume of liquid in mm such as etape. Or does this make the namespace too generic?

Reply:
Both seem fine to me